prow/cmd/jenkins-operator/main.go

```diff
@@ -51,6 +53,10 @@ func main() {
 	flag.Parse()
 	logrus.SetFormatter(&logrus.JSONFormatter{})
 
+	if _, err := labels.Parse(*selector); err != nil {
```

Comment:
Why not pass the result into `NewController` instead of the string?

Reply:
Because all the controller is doing is passing the string through to the api server as a url parameter. I could switch the client to use a labels.Selector instead of a string but it would still need to convert that back to a string.